docs/helpful-guides/first-steps.md

```diff
@@ -0,0 +1,111 @@
+First steps to use Yii2-usuario
+===============================
+
+After installing the extension and having configured everything, you need setup your application with the all the user related stuff, e.g.
+
+* creating first users, roles, permissions, ...
+* assigning permissions and roles to users
+* extending your controllers with access restrictions  
+* starting user management
+
+## Creating your first user
+
+There are several ways to do that:
+
+* using migrations
+* using the Command line [Console Commands](../installation/console-commands.md)
+
+### Creating the first Administrator during a migration
+
+This is helpful e.g. when using you Yii2 applicatio with Docker and need to start the Docker Container with basic user settings.
+
+We will create an ```admin```user with a ```adminisrator``` role. 
+
+Put this in your migration:
+
+    class m... extends Migration
+    {
+        public function safeUp()
```

Comment:
Whilst a migration could make sense, we normally use a `Console` command and allow the user to create any user he wishes to and not being forced to have a hardcoded one as the migration does.

Reply:
yes, the Console command would be the better way.

When you forget to remove/edit/delete is  ,this can be a security issue, when the user expect to use the Console command and an admin user was created before...
Not everyone call his user "admin" ;)